client/my-sites/domains/domain-management/transfer/transfer-page/index.tsx

```diff
@@ -302,6 +346,36 @@ const TransferPage = ( props: TransferPageProps ) => {
 		);
 	};
 
+	const renderUpsellDiscount = () => {
+		return (
+			<>
+				<div key="separator" className="transfer-page__item-separator" />
+				<div className="transfer-page__upsell-discount">
+					<p>
+						{ __( 'Are you sure you want to transfer your domain?' ) }
+						<br />
+						<b>
+							{ sprintf(
+								/* translators: %s is the percentage of the discount */
+								__( 'Renew now and get a %s discount!' ),
+								'10%'
+							) }
+						</b>
+					</p>
+					<RenewButton
+						primary
+						purchase={ purchase }
+						selectedSite={ selectedSite }
+						subscriptionId={ parseInt( domain?.subscriptionId ?? '', 10 ) }
+						tracksProps={ { source: 'test-registered-domain-transfer-out' } }
```

Comment:
```suggestion
						tracksProps={ { source: 'prevent-transfer-out-upsell' } }
```
What do you think of this suggestion?